packages/edit-site/src/components/header/more-menu/style.scss

```diff
@@ -0,0 +1,35 @@
+.edit-site-more-menu {
+	margin-left: -4px;
+
+	// the padding and margin of the more menu is intentionally non-standard
+	.components-button {
+		width: auto;
+		padding: 0 2px;
+	}
+
+	@include break-small() {
+		margin-left: 0;
+
+		.components-button {
+			padding: 0 4px;
+		}
+	}
+}
+
+.edit-site-more-menu__content .components-popover__content {
+	min-width: 260px;
+
+	// Let the menu scale to fit items.
+	@include break-mobile() {
```

Comment:
This is probably copied, but that particular breakpoint is not something we use often, I guess we prefer "small/medium"

Reply:
Do you know the reasons for these styles?

Reply:
Tbh I don't. I just copied the CSS file from the existing component and made a note to circle back and check if all of this is needed. I tried testing on mobile widths but couldn't notice any visual difference.

I removed this in 5582081785d77a9e67a1a1a746af76e0ac59200c but let me know if you'd prefer me to do something else.